grpclib/protocol.py

```diff
@@ -514,6 +596,8 @@ def process_request_received(self, event: RequestReceived) -> None:
         release_stream = self.register(stream)
         self.handler.accept(stream, event.headers, release_stream)
         # TODO: check EOF
+        self.connection.last_headers_received = time.monotonic()
+        self.connection.data_process()
```

Comment:
I think that the same should be done in `process_response_received` and in `process_trailers_received`, they all are called after we receive HEADERS frame.

Reply:
Now it seems to me that the received message and headers do not interest us.
This channel argument controls the maximum number of pings that can be sent when there is no other data (data frame or header frame) **to be sent**

Reply:
Agreed

Reply:
done